kie-server-parent/kie-server-services/kie-server-services-optaplanner/src/main/java/org/kie/server/services/optaplanner/OptaplannerKieServerExtension.java

```diff
@@ -88,12 +88,16 @@ public void init(KieServerImpl kieServer, KieServerRegistry registry) {
         // If new jobs are submitted and all threads are busy, the default reject policy will kick in.
         int availableProcessorCount = Runtime.getRuntime().availableProcessors();
         int resolvedActiveThreadCount = Math.max(1, availableProcessorCount - 2);
+        int queueSize = Integer.parseInt(System.getProperty(
+                KieServerConstants.KIE_OPTAPLANNER_THREAD_POOL_QUEUE_SIZE, String.valueOf(resolvedActiveThreadCount)));
+        logger.info("Creating a ThreadPoolExecutor with corePoolSize = " + resolvedActiveThreadCount + ","
```

Comment:
Is this information important enough to deserve info log level? Wouldn't be enough to use debug level?

Reply:
I'm under impression that component's configuration values deserve INFO level. And also helpful from support point of view.

For example)
~~~
2019-07-10 16:26:36,400 INFO  [org.jbpm.executor.impl.ExecutorImpl] (EJB default - 1) Starting jBPM Executor Component ...
         - Thread Pool Size: 1
         - Retries per Request: 3
         - Load from storage interval: 0 SECONDS (if less or equal 0 only initial sync with storage)
~~~

But if you push again for DEBUG level, it's fine for me to change. Please let me know your thought.

Reply:
Sounds reasonable, lets keep there INFO.

Reply:
@sutaakar ok, I 'll presume you're ok with merging this now?